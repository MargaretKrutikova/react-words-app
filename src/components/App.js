import React from 'react';
import WordsContainer from './words/WordsContainer';

class App extends React.Component {
  render() {
    return (
      <div className="App container">
        <h1>Word list</h1>
        <p>Swedish words are here</p>
        <WordsContainer />
      </div> 
    );
  }
}

export default App;