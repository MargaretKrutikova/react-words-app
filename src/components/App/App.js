import React from 'react';
import WordsContainer from '../Words/WordsContainer';

class App extends React.Component {
  render() {
    return (
      <div className="App container">
        <h1 className="col-xs-12">Word list</h1>

        <div className="col-xs-12 col-sm-12 col-md-6" >
          <WordsContainer />
        </div>
      </div> 
    );
  }
}

export default App;