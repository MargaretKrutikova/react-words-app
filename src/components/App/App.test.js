import 'babel-polyfill'; // required for Object.values

import React from 'react';
//import Enzyme from 'enzyme';
//import Adapter from 'enzyme-adapter-react-16';

import App from './App';

// Enzyme.configure({ adapter: new Adapter() });

// describe('App', () => {
//   it('renders correctly', () => {
//     const wrapper = Enzyme.shallow(<App />);

//     expect(wrapper.children().length).toBe(2);
//     // snapshot expectations
//     expect(wrapper).toMatchSnapshot(); // if anything changes the snapshot expectation will fail
//   });
// });

import renderer from 'react-test-renderer';
test('Link renders correctly', () => {
  const tree = renderer
    .create(<App />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});