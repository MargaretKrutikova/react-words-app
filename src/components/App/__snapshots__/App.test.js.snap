// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<div
  className="App container"
>
  <h1
    className="col-xs-12"
  >
    Word list
  </h1>
  <div
    className="col-xs-12 col-sm-12 col-md-6"
  >
    <div
      style={
        Object {
          "padding": "30px 0",
        }
      }
    />
    <nav
      aria-label="Page navigation"
    >
      <ul
        className="pagination"
      >
        <li
          className="disabled"
        >
          <a
            href="#"
            onClick={[Function]}
          >
            <span>
              First
            </span>
          </a>
        </li>
        <li
          className="disabled"
        >
          <a
            href="#"
            onClick={[Function]}
          >
            <span
              aria-hidden="true"
            >
              «
            </span>
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="#"
            onClick={[Function]}
          >
            <span
              aria-hidden="true"
            >
              »
            </span>
          </a>
        </li>
        <li
          className=""
        >
          <a
            href="#"
            onClick={[Function]}
          >
            <span>
              Last
            </span>
          </a>
        </li>
      </ul>
    </nav>
  </div>
</div>
`;
